src/lib/renderComponent.tsx

```diff
@@ -73,7 +73,11 @@ const renderComponent = <P extends {}>(
         renderer = felaRenderer,
       }: IThemeInput | IThemePrepared = {}) => {
         const ElementType = getElementType({ defaultProps }, props)
-        const rest = getUnhandledProps({ handledProps }, props)
+        const accessibility = getAccessibility(props, state)
+        const rest = getUnhandledProps(
```

Comment:
this introduces tough coupling between rendering logic and accessibility. Let me, please, take another look on task that is aimed to achieve

Reply:
to me it seems more reasonable to provide an approach that is based on component's anatomy - the same way how styles are applied for component's children:

```jsx
const styles = {
  root: <root styles>,
  anchor: <anchor styles>
}

// and then applied in the following way
render({ classes }) {
  return (
    <ElementType classNames={classes.root} .. >
        <a classNames={classes.a} ... />
    </ElementType>
  )
)
```

For behaviors I would suggest to introduce the same idea:

```jsx
const Menu: Accessibility = (props: any) => ({
  attributes: {
    root: {
      ...
    },
    itemAnchor: {
      ...
    }
  },
})
```

Reply:
Left the implementation as is, after our discussions/.